main.go

```diff
@@ -12,36 +14,69 @@ import (
 // JSON example
 // {"ar": ["joaothallis"], "elixir": ["josevalim", "eksperimental"]}
 
-func main() {
-	dir := os.Args[1]
-	homeDir, err := os.UserHomeDir()
+func getPrarFilePath() (string, error) {
+	globalFlag := flag.Bool("global", false, "When you choose to using ./config/prar.json")
+	flag.Parse()
+	if *globalFlag {
+		homeDir, err := os.UserHomeDir()
+		return homeDir + "/.config/prar.json", err
+	} else {
+		return "./.prar.json", nil
+	}
+}
 
-	filePath := homeDir + "/.config/prar.json"
+func getProjectName() (string, error) {
+	args := flag.Args()
+	if len(args) >= 1 {
```

Comment:
I think this is more common to verify strings
```suggestion
	if len(args) != "" {
```

Reply:
the flag.Args() function return the string list `string[]` and not `string`